build_data.js

```diff
@@ -673,6 +673,17 @@ function translationsToYAML(translations) {
 }
 
 
+function writeRapidConfig() {
+    var readRapidConfig = readFileProm('data/rapid_config.yaml', 'utf8'); 
```

Comment:
will this break if `data/rapid_config.yaml` doesn't exist?

Reply:
This line will not break at build time- the line that will break is actually the import of rapid_config.json inside `data/index.js`.